Detector/DetStudies/src/components/FilterSiliconEcalHits.cpp

```diff
@@ -0,0 +1,107 @@
+
+#include "FilterSiliconEcalHits.h"
```

Comment:
Is this filter used in the chain of sim->digi->reco?
If so, maybe it should be moved to `DetComponents`.
If that is just for the analysis of the calo hits, however, it may stay here.